packages/ee/ui-platform/src/lib/lib.routes.ts

```diff
@@ -49,6 +49,9 @@ export const uiEePlatformRoutes: Route[] = [
         data: {
```

Comment:
Does routes updates when the tab changes? Is it possible to link the tab directly in the docs?